doc/configuration.md

```diff
@@ -83,9 +83,12 @@ limits.processes            | integer   | - (max)       | yes           | Maximu
 linux.kernel\_modules       | string    | -             | yes           | Comma separated list of kernel modules to load before starting the container
 raw.apparmor                | blob      | -             | yes           | Apparmor profile entries to be appended to the generated profile
 raw.lxc                     | blob      | -             | no            | Raw LXC configuration to be appended to the generated one
+raw.seccomp                 | blob      | -             | no            | Raw LXC seccomp profile string to append to the generated one
 security.nesting            | boolean   | false         | yes           | Support running lxd (nested) inside the container
 security.privileged         | boolean   | false         | no            | Runs the container in privileged mode
 user.\*                     | string    | -             | n/a           | Free form user key/value storage (can be used in search)
+security.syscalls.default   | boolean   | true          | no            | Enables the default syscall blacklist
+security.syscalls.compat    | boolean   | true          | no            | On x86\_64 this enables blocking of compat\_\* syscalls, it is a no-op on other arches
```

Comment:
Hey - thanks for doing this.  But a few comments:

1. it's not clear to me whether 'security.syscalls.compat = true' means allow them or block them.
2. is just appending a raw.seccomp always enough?  Or does raw.seccomp really need to be a replacement?  I would have actually expected a security.syscalls.whitelist and blacklist which would get turned into a seccomp config for you, and raw.seccomp finally to be a full replacement, ie. security.syscalls.* is ignored if raw.seccomp is set.